src/OpenLoco/Objects/TrainStationObject.h

```diff
@@ -59,8 +59,8 @@ namespace OpenLoco
 
     namespace TrainStation::ImageIds
     {
-        constexpr uint32_t preview_image = 0;
-        constexpr uint32_t preview_image_windows = 1;
+        static constexpr uint32_t kPreviewImage = 0;
+        static constexpr uint32_t kPreviewImageWindows = 1;
```

Comment:
Why were these made static?

Reply:
code style - everything else being made constexpr is also static. should I not make it static?

Reply:
I found other ImageIds that weren't static, so I think I'll revert the static here